spec/features/gardens/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Index Page' do
+  describe 'a logged in user' do
+    before :each do
+      @user = User.new({id: 1,
+                      attributes: {
+                          email: '123@gmail.com' },
+                      relationships: {
+                          gardens: {
+                              data: [ { id: 1,
+                                        attributes: {
+                                            name: 'My Garden',
+                                            latitude: 23.0,
+                                            longitude: 24.0,
+                                            description: 'Simple Garden',
+                                            private: false }} ] }}})
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+    end
+  end
+end
```

Comment:
Little confused about this. Is this just an unfinished test?

Reply:
Yeah I'm not sure what this is. We don't even have a gardens index_page that I know of. Good catch.

@D-Lessenden any insight on this?